### PR TITLE
Add touch feedback to Facia card links

### LINKED ISSUE #9960: No feedback when tapping Facia cards on mobile
When you click a link on mobile, you don't get any feedback before the browser navigates to the page. Basic a11y…

/cc @rich-nguyen

### PR DESCRIPTION
**Pull request by OliverJAsh**
Fixes #9960

If we can do this, I'm happy to remove FastClick, re. #9947

/cc @piuccio

### CODE REVIEW

**Comment by piuccio** [comment]
Looks good :+1: 

Let me throw some considerations, if you have some time you can verify them and decide whether there's any truth in them.

1. You're using `mouseenter` and `leave`. Those two events were introduced in IE and later adopted by other browsers, because they react on mouse events, they only make sense on desktop browsers. This means it's equivalent to the `:hover` selector. Old IE does not support `:hover` on non clickable elements, but IE8 does. Mobile browsers convert the first tap to a hover state, but only if the element is clickable. In this case, because you apply the `:hover` to `$('.u-faux-block-link__overlay').parent()` which is a DIV, you shouldn't have a problem.
 
 tl;dr it's probably fine to replace `mouseenter` with pure CSS.


2. Giving this a second thought, maybe `touchend` is not enough, if you touch an element, then drag the finger and release when you're on another card, what events are fired? Do you have a `touchend` on the same element that received the `touchstart`? If not you should either attach a `touchcancel` (if that's fired at all) or store a reference of the highlighted element in `touchstart` and then remove the class from that element on `touchend`.

let's get rid of fastclick!!

**Comment by heralder99** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

heralder@naver.com 님께 보내신 메일 <Re: [frontend] Add touch feedback to Facia card links (#10046)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Comment by OliverJAsh** [comment]
Re. point 1, as I mentioned when we spoke in person, I'm not entirely sure why we're using `mouse{enter,leave}` instead of simple CSS `hover` states. These "faux block links" are sort of a hack, so I suspect there is a good reason for it. I started to refactor `u-faux-block-link` to use `hover` states but it's quite a big piece of work because its usage is widespread (breaking news, rich links, different types of facia cards), and that's not worth doing.

> if you touch an element, then drag the finger and release when you're on another card, what events are fired?

Wouldn't your finger be on the same item, because the page moves with your finger?

**Comment by piuccio** [comment]
> Wouldn't your finger be on the same item, because the page moves with your finger?

Maybe not when you move horizontally on a tablet (because on mobile cards take the whole width)

But I've given my :+1:  already, so you can ship it as it is and then eventually fix it if there's really a problem.

**Comment by OliverJAsh** [comment]
> Maybe not when you move horizontally on a tablet (because on mobile cards take the whole width)

I gave this a quick test: http://output.jsbin.com/wenuha/5

Seems like `touchstart` will always fire for the same element as `touchend`.

**Comment by piuccio** [comment]
alert says `true` also on my mobile
